### PR TITLE
Flybloc Grant Application

### PR DESCRIPTION
**Pull request by FlyblocDev**
# Grant Application Checklist

- [x] The [`applications/application-template.md`](../applications/application-template.md) or [`applications/research-template.md`](../applications/research-template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] An Ethereum address for the payment of the milestones is provided inside the application.  
- [x] The total funding amount of the project is up to USD $30k for every grant applications.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).

### CODE REVIEW

**Comment by libotony** [comment]
Thank you for moving your application to the appropriate folder. However, you are still using the template for the general grant. Could you please adjust your application to align with the [EasyA Grant Track template](https://github.com/vechain/grant-program/blob/90ca36f2c30758b4690ceaf9ff3ef9da9f4363fa/Vechain-EasyA-Hackathon-Boston-Oct-2023/applications/FibreTag.md)?

Please avoid opening a new pull request if possible. Instead, submit a new commit within this existing pull request to update your application. Thank you for your cooperation.

File: applications/Flybloc.md [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+
+## Architecture
+Leveraging Unity3D as our cornerstone, the design encapsulates a primary UI/UX app interfacing with a mobile web full-stack. This ensures a smooth transition and interaction with various game modules. Integration with the Supabase API forms the backbone of our real-time score tracking and user data analytics. The games, though diverse, share common attributes of being engaging and easy to grasp. On the blockchain front, the platform is crafted to reward users for their interactions, with tokens that can be converted into tangible, brand-centric benefits. To top it off, our approach is built on simplicity, ensuring users engage with the brand and its rewards without being entangled in the intricacies of blockchain mechanics.
+
+Please include the team's long-term project plans, operation plans and intentions.
+
+These are the deliverables we submitted for the Harvard 2024 VeChain Hackathon. 
+
+- [Project’s one-pager](https://drive.google.com/file/d/1-XyiktDbbNv989igzI_MBvY1fkbqWTZf/view?usp=drive_link) (provided by BCG at the Hackathon)
```
Comment (libotony) [inline applications/Flybloc.md:31]: I have requested document access for the account `tony.li@vechain.org`, please kindly check your inbox and give read access to that account.

**Review by libotony — CHANGES_REQUESTED** [review]
Please adjust your application to align with the [EasyA Grant Track template](https://github.com/vechain/grant-program/blob/90ca36f2c30758b4690ceaf9ff3ef9da9f4363fa/Vechain-EasyA-Hackathon-Boston-Oct-2023/applications/FibreTag.md)

**Comment by libotony** [comment]
@FlyblocDev Thank you for the update, but your application seems to be in the folder of the general grant application. Please consider moving to folder [Vechain-EasyA-Hackathon-Boston-Oct-2023](https://github.com/vechain/grant-program/tree/master/Vechain-EasyA-Hackathon-Boston-Oct-2023/applications).

**Comment by NecoSherry** [comment]
Thank you for your interest in our grant program and for submitting your application. We regret to inform you that your application has been rejected by the grant committee. 

The main reason is that your application lacks sufficient description regarding how it supports brands and its profitability through its business model. 

I’m therefore closing the PR.  In any case, we thank you for your interest in the grant program.

**Pull request closed without merging**